package.json

```diff
@@ -32,8 +32,12 @@
     "swagger-ui-express": "^5.0.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.23.3",
+    "@babel/preset-env": "^7.23.3",
+    "@babel/preset-typescript": "^7.23.3",
     "@types/config": "^3.3.2",
     "@types/express": "^4.17.19",
+    "@types/jest": "^29.5.8",
     "@types/lodash": "^4.14.200",
```

Comment:
If you do not already use lodash library, you need to remove it from devDep and dependencies

Reply:
Lodash library has been removed from dependencies.